README.md

```diff
@@ -42,8 +42,15 @@ This part describes the Docker image that this repo is designed to build. People
 - `MW_RECAPTCHA_SITEKEY`
 - `MW_RECAPTCHA_SECRETKEY`
 - `PLATFORM_API_BACKEND_HOST`: points to an internal mode wbstack api service
-- `MW_ELASTICSEARCH_HOST`: elasticsearch hostname
-- `MW_ELASTICSEARCH_PORT`: elasticsearch port
+- `MW_ELASTICSEARCH_HOST`: DEPRECATED: elasticsearch hostname. If set "default" settings below are ignored.
+- `MW__ELASTICSEARCH_PORT`: DEPRECATED: elasticsearch hostname
+- `MW_DEFAULT_ELASTICSEARCH_HOST`: default (i.e. where data is served to users from) elasticsearch hostname
+- `MW_DEFAULT_ELASTICSEARCH_PORT`: default elasticsearch port
+- `MW_DEFAULT_ELASTICSEARCH_ES6`: use ES6 compatability layer with default cluster
```

Comment:
maybe we should describe it in the way that it is obvious it is a boolean.

Reply:
Done!